cyences_app_for_splunk/default/macros.conf

```diff
@@ -1232,6 +1232,9 @@ iseval = 0
 definition = search *
 iseval = 0
 
+[cs_windows_defender_malware_detected_local_time_converter]
```

Comment:
That detection time field is already human readable.
Since we have to convert it to local time zone we are first parsing it and then converting it to local time, that why we are using this new macro